### PR TITLE
Removed restrictions on LASYM values (NJOY uses more than just 0 and 1)

### PR DESCRIPTION
**Pull request by whaeck**

### CODE REVIEW

File: src/ENDFtk/section/7/4/src/ctor.hpp [hunk 1]
```diff
@@ -27,11 +27,10 @@ Type( double zaid, double awr, int lat, int lasym,
   verifySecondaryTemperatures( this->b_.analyticalFunctionTypes(),
                                this->secondary_ );
   checkValue( this->LAT(), "LAT" );
-  checkValue( this->LASYM(), "LASYM" );
```
Comment (nathangibson14) [inline src/ENDFtk/section/7/4/src/ctor.hpp:30]: Why are you getting rid of the check rather than fixing the check?

**Review by nathangibson14 — COMMENTED** [review]
Hard to find the real changes in the sea of whitespace tweaks. :)

Looks fine to me overall, just left one question.

File: src/ENDFtk/section/7/4/src/ctor.hpp (hunk 1 above)
Comment (whaeck) [reply src/ENDFtk/section/7/4/src/ctor.hpp:30]: I don't know which values are being used internally by NJOY.

**Comment by whaeck** [comment]
@nathangibson14 That's my editor, it automatically removed trailing white space.

File: src/ENDFtk/section/7/4/src/ctor.hpp (hunk 1 above)
Comment (whaeck) [reply src/ENDFtk/section/7/4/src/ctor.hpp:30]: I can add the few I know, but we'll have to revisit this whenever something else comes up. It shoudl be noted that LASYM has no effect on the structure of the file.

**Review by jlconlin — APPROVED** [review]

**Pull request merged**